task.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+mkdir task
+# создаём каталог task с вложенными директориями
+# task
+#   dir1
+#   dir2
+#   dir3
+#       dir4
+mkdir ./task/dir1 ./task/dir2 ./task/dir3 ./task/dir3/dir4
```

Comment:
Можно не указывать везде `./`, кроме 35 строчки

Так же, чтобы не создавать изначально папку task и промежуточные можно использовать флаг `-p`

```suggestion
mkdir -p task/dir1 task/dir2 task/dir3/dir4
```